### DISCUSSION TITLE
New logo and standard theme

### DISCUSSION THREAD

**Discussion by StigtriX** [sumatrapdfreader/sumatrapdf#3251] [Ideas]
### I have a suggestion for a standard theme where I have done some small updates for a cleaner look:

- New logo that looks like a stack of paper
- Font used for logo and text: Open Sans
- Blue theme colors
- New icons based on Microsoft Fluent Emoji (open source)
- Icons for file types are not there by default (looks messy)
- Wording changed from "Frequently read" to "Favorites" (would also like more entries, in line with "Home Page Favorites #3077")
- Soft shadows added to images
- Sumatra logo and version number moved to the bottom, leaving more room for- and focus on books
- Text with links are not underlined, but the text changes color when you hover above it with your cursor
![Sumatra PDF - New logo, UI   Dark theme](https://user-images.githubusercontent.com/58557127/216632894-fbcfb137-e50e-4068-9328-823186613517.png)

**Comment by StigtriX** [comment]
EDIT: added a suggestion for a dark theme.

**Reply by GitHubRulesOK** [reply]
nice but how are you doing that "small update" in Visual Studio for using Sumatra code?, a few have tried and failed.
egg shell blue is not a problem but the heading is 3rd party theming (use blinds)
![image](https://user-images.githubusercontent.com/12754913/216652552-d9ddf7f2-1128-4ba8-b868-f3722348e5ea.png)

The bigger problem is with black you need different set of thumbnails to the normal ones even the inverted one (bottom right) does not show. needing a whole rewrite with rainbow of thumbnails. and text colours  

![image](https://user-images.githubusercontent.com/12754913/216652915-34bc1f49-3d20-45e6-b405-669c4804738a.png)

**Reply by StigtriX** [reply]
> nice but...

I have not experience with Visual Studio. This is a Figma mock-up. Do you have a recommendation for a place to start if I want to start with Visual Studio and have a gander at the challenges?

**Reply by GitHubRulesOK** [reply]
it is not trivial as SumatraPDF was developed with XP  (pre metro) colour definitions and although XP is no longer supported there are Many win7 users even if that too is EOL, in reality the whole question of theming needs a fresh UI container  and embed the core viewer inside, this has actually been done by the cloners of SumatraPDF but they charge around $100 a copy to pay for their hoards of "technical support" staff. If I had 1c for every answer  I would be rich too.

**Reply by StigtriX** [reply]
That is a shame... : / The functionality and speed is awesome, but the UI and UX look old.


**Would it be possible to just do a few updates instead then?**

1. Remove the bar with the colorful logo and version number completely
2. Remove underlines from the clickable text
3. Change the name from Frequently Read to Favorites

I think it looks a lot cleaner now.
![Sumatra - Slight Update](https://user-images.githubusercontent.com/58557127/218571814-d391aafc-62a9-4428-91d1-fcd2132adfda.png)

**Reply by GitHubRulesOK** [reply]
you are mixing up favorites (unlimited page references) with 10 most frequently read/pinned documents (those are totally different user functions)
![image](https://user-images.githubusercontent.com/12754913/218579623-4611b2ac-9df5-4b70-b779-38314ec9728d.png)


Personally I depend on the version number more frequently than anything else on that page, it confirms before launch if its todays version.

Perhaps move the others from bottom up to top would be more useful to generate less divisions, but its not my interface.

**Reply by StigtriX** [reply]
Thank you for the clarification. That makes sense.

A good suggestion. Could look like this:
![After v2](https://user-images.githubusercontent.com/58557127/218580861-e049e429-c309-4c80-aabf-c88965628233.png)

**Comment by kjk** [comment]
I would like to improve the UI. Sadly, making seemingly simple color changes require lots of effort due to using mostly win32 API, which don't offer easy ways to change colors.

Moving logo from top to bottom might make sense.

It's not really `Favorites` list. The current name `Frequently Read` conveys how that page is constructed: it weights frequently opened document more and displays them first.

This might frankly be too complicated. I might go with "most recent first" in which case a proper name would be `Recently opened`.

**Reply by StigtriX** [reply]
I understand. It is by far the most fluid PDF tool I have used, so this is just hopes and nitpicking from me 😃
